rsciio/utils/tools.py

```diff
@@ -516,3 +516,21 @@ def get_file_handle(data, warn=True):
                     "an hdf5 file."
                 )
     return None
+
+
+def jit_ifnumba(*args, **kwargs):
+    try:
+        import numba
+
+        if "nopython" not in kwargs:
+            kwargs["nopython"] = True
+        return numba.jit(*args, **kwargs)
+    except ImportError:
```

Comment:
We could add a logger warning here to explain that the code is not running with optimal performance because number is not installed.